.travis.yml

```diff
@@ -1,34 +1,40 @@
 language: php
 
-dist: precise
+dist: trusty
 
-sudo: false
+cache:
+  directories:
+    - $HOME/.composer/cache/files
 
 php:
   - 5.6
   - 7.0
   - 7.1
 
 env:
-  - DB=SQLITE CORE_RELEASE=4 PDO=1
+  global:
+    - DB=SQLITE
+    - PDO=1
 
 matrix:
   fast_finish: true
   include:
     - php: 5.6
-      env: DB=SQLITE CORE_RELEASE=4 PDO=0
+      env: PDO=0 PHPCS_TEST=1
 
 before_script:
   # Init PHP
-  - printf "\n" | pecl install imagick
   - phpenv rehash
   - phpenv config-rm xdebug.ini
+  - export PATH=~/.composer/vendor/bin:$PATH
   - echo 'memory_limit = 2048M' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
 
 # Install composer dependencies
   - composer validate
-  - composer require symfony/config:^3.2 silverstripe/framework:4.0.x-dev silverstripe/cms:4.0.x-dev silverstripe/siteconfig:4.0.x-dev silverstripe/config:1.0.x-dev silverstripe/admin:1.0.x-dev silverstripe/assets:1.0.x-dev silverstripe/versioned:1.0.x-dev --no-update
+  - composer require --prefer-dist --no-update silverstripe/recipe-cms:1.0.x-dev
```

Comment:
`--prefer-dist` here does nothing

Reply:
Not all dependencies of recipe-cms are dev branches. E.g. symfony.

Reply:
Yes, but when using `--no-update` only the composer file is modified and no installs happen. There is no meta data stored that means when you subsequently run an install that `--prefer-dist` will be used.

The next line runs the install and correctly uses the `--prefer-dist` flag